README.md

```diff
@@ -63,7 +63,7 @@ import PostShowPage from './components/PostShowPage'
 import PostEditPage from './components/PostEditPage'
 
 export default class Router extends SimpleReactRouter {
-  getRoutes({map, props, redirectTo}){
+  getRoutes(map, props, redirectTo){
```

Comment:
I generally prefer named arguments to ordered arguments. Was this change necessary other than style preference?